cmake/build_mingw32.sh

```diff
@@ -18,6 +18,8 @@ fi
 
 if [ $QT5 ]; then
 	CMAKE_OPTS="-DWANT_QT5=$QT5 -DCMAKE_PREFIX_PATH=$MINGW $CMAKE_OPTS"
+else
+	CMAKE_OPTS="-DWANT_QT5=OFF"
```

Comment:
What does this `else` achieve?

Reply:
Ok, I think I understand it... it sets the Cmake flag back to it's old default value... but then we still need "Qt5 instructions in the wiki", since this flag is needed for Windows builds, no?  I guess that's an OK compromise.  Any alternative attempt would break backwards compat on build systems.

Reply:
@tresf Sorry for te very late reply. If I recall correctly, I didn't want to touch the travis scripts which all use the environment variable `QT5=True` for Qt5 and `QT5=` (unset) for Qt4. I didn't consider that `build_mingw32.sh` is not just used internally, so yeah... It'd make sense to change this to be consistent, but it would also require reversing the Travis Qt4/Qt5 env variable logic.